Esri/arcgis-maps-sdk-swift-samples
# [New] Create symbol styles from web styles

## Description

This PR implements `Create symbol styles from web styles` in `Visualization` category.
URL to README: [ReadME](https://github.com/Esri/arcgis-maps-sdk-swift-samples/tree/Caleb/New-CreateSymbolStylesFromWebStyles/Shared/Samples/Create%20symbol%20styles%20from%20web%20styles)

## Linked Issue(s)

- `swift/issues/4071`

## How To Test

- Pan the map and zoom the map to see the symbols update once they load.
- Tap the "Legend" button to open the legend.

## Screenshots

![create-symbol-styles-from-web-styles](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/34120440/09cf61b8-30ac-494f-b7e7-9310719d82c3)

## Discussion

- Should this sample include a status message overlay for the symbols/swatches loading similar to the [one in the Display content of utility network](https://github.com/Esri/arcgis-maps-sdk-swift-samples/blob/main/Shared/Samples/Display%20content%20of%20utility%20network%20container/DisplayContentOfUtilityNetworkContainerView.swift#L55-L60) sample?
- This is my first sample back, so if you see anything that you think is off, I probably just forgot something so please let me know!
